<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>381</width>
      <height>521</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>381</width>
      <height>521</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="EditPlan" name="EditPlan"/>
    <widget class="EditPlot" name="EditPlot"/>
    <widget class="EditRoutine" name="EditRoutine"/>
    <widget class="EditSetting" name="EditSetting"/>
   </widget>
   <widget class="QPushButton" name="BtnOk">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>560</y>
      <width>94</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnCancel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>560</y>
      <width>94</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnSave">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>560</y>
      <width>94</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="TreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>301</width>
      <height>541</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="animated">
     <bool>true</bool>
    </property>
    <property name="expandsOnDoubleClick">
     <bool>false</bool>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>108</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="BtnAddPlot">
       <property name="text">
        <string>Add Plot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnAddRoutine">
       <property name="text">
        <string>Add Routine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnAddLimit">
       <property name="text">
        <string>Add Limit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnAddSetting">
       <property name="text">
        <string>Add Setting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnAddRoute">
       <property name="text">
        <string>Add Route</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnAddTrace">
       <property name="text">
        <string>Add Trace</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnDel">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnMultiEdit">
       <property name="text">
        <string>Multi edit</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnExpandAll">
       <property name="text">
        <string>Expand all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnCollapseAll">
       <property name="text">
        <string>Collapse all</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbEditor">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>55</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Editor</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>135</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="BtnAddItem1">
       <property name="text">
        <string>Add Device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnAddItem2">
       <property name="text">
        <string>Add Limit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnAddItem3">
       <property name="text">
        <string>Add Line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnAddItem4">
       <property name="text">
        <string>Add InstructionFile</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnDelItem">
       <property name="text">
        <string>Del Item</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditPlan</class>
   <extends>QWidget</extends>
   <header>EditPlan.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditPlot</class>
   <extends>QWidget</extends>
   <header>EditPlot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">TreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>EditRoutine</class>
   <extends>QWidget</extends>
   <header>EditRoutine.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditSetting</class>
   <extends>QWidget</extends>
   <header>EditSetting.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
